virttest/utils_misc.py

```diff
@@ -4378,3 +4378,34 @@ def _md5(fd):
         if _md5(fd_a) == _md5(fd_b):
             return True
     return False
+
+
+def is_linux_uefi_guest(runner=cmd_status_output):
+    """
+    Check whether guest is uefi guest
+    """
+    cmd = "ls /sys/firmware/efi"
+    status, output = runner(cmd)
```

Comment:
You're not using 'output', you can omit it, `status, _ = runner(cmd)`.

Reply:
Valid nitpick, changes made.